### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by noelfoka**
In this branch, I did create main.js file and edit it.

### CODE REVIEW

**Review by m05e5 — CHANGES_REQUESTED** [review]
Hi @noelfoka  , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, where ESLint check is not running. Kindly use the linters.yml file from the [HTML-CSS-JS](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) folder of the linters config repository.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -17,12 +16,29 @@
...
+                            id="menu-icon" />
+                    </a>
+                    <!-- mobile menu -->
+                    <div class="mobile-menu-container mobile-menu-toggler">
+                        <div class="menu-icon-container">
+                            <a><img
+                                    src="./images/mobile_exit_Icon.png"
+                                    alt="exit icon"
+                                    id="exit-menu-icon"
+                                    /></a>
```
Comment (Strangeal) [inline index.html:30]: Kindly consider taking a second look at your `images` folder as there seems to be no `image` with the name `mobile_exit_Icon.png` due to your close image isn't displaying properly on your page.

Screenshot 2023-04-17 at 17 13 50

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @noelfoka and @hagan199, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -61,6 +64,57 @@ body {
   transform: scale(1.2);
 }
```
Comment (amejid) [inline styles.css:66]: - [ ] [OPTIONAL] The navigation links are not showing on the desktop version. I highly suggest you attend to this issue. 🥸️

image

**Review by amejid — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct use of Github flow :heavy_check_mark: 
- Mobile menu working as expected :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**